slick/src/main/scala/slick/ast/Comprehension.scala

```diff
@@ -6,17 +6,19 @@ import slick.util.ConstArray
 
 /** A SQL comprehension */
 final case class Comprehension(sym: TermSymbol, from: Node, select: Node, where: Option[Node] = None,
+                               updateValues: ConstArray[Node] = ConstArray.empty,
```

Comment:
Why is `updateValues` an array? `select` is a single node at this point. Since Comprehensions only show up after the flattening phases, it should always be a `StructNode` or `ProductNode` (depending on the phase). Shouldn't `updateValues` be the same?